tests/unit/evaluation/SplittingStrategy_unittest.cc

```diff
@@ -296,3 +296,90 @@ TEST(SplittingStrategy,LOO)
 		SG_UNREF(splitting);
 	}
 }
+
```

Comment:
I am missing a test where you do things/checks by hand

 * manually specify some simple data
 * manually explicitly check the folds (since you know the data)
 * really small example, yet very useful